src/legacy/core_plugins/data/public/plugin.ts

```diff
@@ -97,11 +99,17 @@ export class DataPlugin implements Plugin<DataSetup, DataStart, DataPluginSetupD
       query: this.query.setup(),
       search: this.search.setup(savedObjectsClient),
     };
+
+    return this.setupApi;
   }
 
   public start(core: CoreStart, plugins: DataPluginStartDependencies) {
     return {
-      expressions: this.expressions.start({ inspector: plugins.inspector }),
+      ...this.setupApi!,
+      expressions: {
+        ...this.setupApi!,
```

Comment:
Why are you destructuring setupApi into expressions?

Reply:
My bad, I meant to spread `...this.setupApi!.expressions`

Reply:
I don't think you should.
They have different APIs ATM.
If you need the same functionality, that should be a change in the expressions sub-service.

Reply:
I don't need it, just thought I put it in there for completeness sake. I will remove it for now.